src/ResourceManager/AnalysisServices/Commands.AnalysisServices.Dataplane/Commands/Get-AzureAsInstanceLog.cs

```diff
@@ -0,0 +1,182 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.Linq;
+using System.Management.Automation;
+using System.Management.Instrumentation;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Models;
+using Microsoft.Azure.Commands.AnalysisServices.Dataplane.Properties;
+using Microsoft.Azure.Commands.Common.Authentication.Models;
+using Microsoft.IdentityModel.Clients.ActiveDirectory;
+using Microsoft.WindowsAzure.Commands.Utilities.Common;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Azure.Commands.AnalysisServices.Dataplane
+{
+    /// <summary>
+    /// Cmdlet to log into an Analysis Services environment
+    /// </summary>
+    [Cmdlet("Get", "AzureAnalysisServicesInstanceLog", SupportsShouldProcess = true)]
+    [Alias("Get-AzureAsInstanceLog")]
+    [OutputType(typeof(bool))]
+    public class GetAzureAnalysisServerLog : AzurePSCmdlet
+    {
+        private string serverName;
+
+        [Parameter(Mandatory = true, HelpMessage = "Name of the Azure Analysis Services which log will be fetched")]
+        [ValidateNotNullOrEmpty]
+        public string Instance { get; set; }
+
+        [Parameter(Mandatory = false)]
+        public SwitchParameter PassThru { get; set; }
```

Comment:
@AOrazaev `PassThru` should not be used for a Get cmdlet, since a Get should _always_ return an object, the default behavior should not be to return nothing.

Was this cmdlet intended to return anything? If not, we should consider changing the verb to something else that accurately represents what this cmdlet does.